### ISSUE TITLE
[FEAT]: add Machine learning interoperability

### ISSUE THREAD

**Issue by thebino** [photos-network/core#32]
### Description

Deep learning and Data mining are mandatory in image recognition and processing today, with convolutional neural networks imagery and pixel data can be analyzed and categorized in a glimpse.

 * The official [tensorflow crate](https://crates.io/crates/tensorflow) should be a good start
 * [aperture](https://github.com/mstallmo/aperture) is specialized on object detection
 * [Awesome-Rust-MachineLearning](https://github.com/vaaaaanquish/Awesome-Rust-MachineLearning) contains lots of resources on that topic

**Reinforcement learning** needs to be covered as well to take care of the user inputs

**Comment by nhhung1810** [comment]
Hi @thebino , do you still need helps on this issues? I'm interested in this, love to talk more about this

**Comment by thebino** [comment]
@nhhung1810 yes, this will be a major task of this application, so every support is welcome.
you can contact me on [Mastodon](https://mastodon.cloud/@photos)

**Comment by nhhung1810** [comment]
I can not use that link, but I send you a invitation through LinkedIn btw
